### ISSUE TITLE
Plot issue when using int64 values

### ISSUE THREAD

**Issue by tkmaker** [h2oai/wave#496]
#### Wave SDK Version, OS

0.10.0

Hit a but while experiment with this code:
```
from h2o_wave import site, data, ui, app, main, Q
import numpy as np

@app('/')
async def serve(q: Q):
    age = [10, 10, 20, 30, 40, 20]
    unique, counts = np.unique(age, return_counts=True)
    rows1 = list(zip(unique, counts))
    rows = [(0,1), (2,3)]
    q.page['plot1'] = ui.plot_card(
        box='1 1 -1 -1',
        title='Age Count',
        data=data('age count', rows=rows1),
        plot=ui.plot([ui.mark(type='interval', x='=age', y='=count', x_min=0, y_min=0,
                              x_title='Age', y_title='Count', color='#33BBFF')]
                     ))
    await q.page.save()
```

If I use rows, the code works fine because the values are int. If I use rows1, the values are of type int64 and it complains about int64 values not being JSON serializable. 

`TypeError: Object of type int64 is not JSON serializable`

Should we try to cast to int to avoid hard fails? 

cc @lo5 @mturoci

**Comment by tkmaker** [comment]
eg. this works as a workaround:
rows1 = [list(zip(unique.to_list(), counts.to_list()))]

**Comment by lo5** [comment]
This is not a bug. In general, we use Python's `json` to serialize values, and `np.int64` is technically not a Python number, therefore `json` will complain.

For now the `to_list()` is necessary, but we should probably look at providing a global switch that optionally turns on numpy-special-handling and deals with such cases.

**Comment by tkmaker** [comment]
Right, I phrased the title incorrectly. This is more about handling this case vs an actual bug.

**Comment by lo5** [comment]
Related to #447

**Comment by ajjayymahato** [comment]
Is this issue open? Can I take up this issue? If yes can you please assign this to me?

**Comment by mturoci** [comment]
Hi @ajjayymahato! You can try it if you feel like it, but it's not going to be easy. I would recommend picking something easier to start - see https://github.com/h2oai/wave/labels/good%20first%20issue.
